packages/bson-bench/src/common.ts

```diff
@@ -44,7 +46,7 @@ export class Package {
       this.library = match[1] as 'bson' | 'bson-ext';
 
       this.localPath = match[2];
-      this.computedModuleName = `${this.library}-local-${this.localPath.replaceAll(path.sep, '_')}`;
+      this.computedModuleName = `${this.library}-local-${this.localPath.replaceAll(sep, '_')}`;
```

Comment:
sep may be the system separator but in practice windows works with both / and \ paths, so this won't work on a windows system where the path has been made with /, should we just always "sanitize" both?